api/src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -88,6 +88,9 @@ class TransactionalEmail(Enum):
     USER_REQUEST_DELETE_ACCOUNT_RECEPTION = models.Template(
         id_prod=511, id_not_prod=54, tags=["reception_demande_suppression_compte_jeune"]
     )
+    NOTIFICATION_BEFORE_DELETING_SUSPENDED_ACCOUNT = models.Template(
+        id_prod=1004, id_not_prod=144, tags=["notification_avant_suppression_compte_suspendu"]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏On a bien vérifié avec la comm que ce tag est bon ?

Reply:
effectivement ce n'est pas le bon je pense que je n'ai pas bien commit la correction

Reply:
je ne sais pas quel tag mettre ce n'est pas dans le ticket à qui demander?